### PR TITLE
English: contact

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translates the contact page

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -82,6 +82,24 @@ en:
     desc: "Performance support through knowledge-on-demand. Enhance your employees’ achievements by providing them with just-in-time knowledge. Keep record of knowledge in a simple manner and share manuals, protocols or work instructions."
     logo: logos/capp-agile-white.png
   contact:
+    heading: "Hi! We’re so happy to meet you"
```
Comment (sn3p) [inline locales/en.yml]: `so`? Are we really THAT HAPPY? :D

**Alternatives:**
`Hi! We're happy to meet you`
`Hi, nice to meet you!`

File: locales/en.yml [hunk 2]
```diff
@@ -82,6 +82,24 @@ en:
...
+    heading2: "We’re here to help, provide you with more information and answer your questions."
+    main:
+      heading: "How can we help?"
+      question:
+        heading: "More information"
+        info: "Do you have a question or would you love to know more about our software and services?"
```
Comment (sn3p) [inline locales/en.yml]: `like to know`?

File: locales/en.yml (hunk 1 above)
Comment (sdegeus) [inline locales/en.yml]: Yeah ' nice to meet you' is better!

File: locales/en.yml (hunk 2 above)
Comment (sdegeus) [inline locales/en.yml]: We'll turn it into like :-)

**Pull request merged**